README.md

```
@@ -2,14 +2,12 @@
 
 # ⚡ QARBI Protocol
 
-### Autonomous AI Agent Civilization & Conway State Engine on Arbitrum Sepolia
-**Powered by Arbitrum Stylus (Rust VM) · NIST FIPS 204 (ML-DSA-65) PQC · Gemini 2.0 Flash**
+### Autonomous AI Agent Prototype & Conway State Engine
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)